whats_fresh/whats_fresh_api/templates/entry.html

```diff
@@ -3,38 +3,47 @@
 {% block content %}
 <div class='entry'>
     <a href='{% url 'list-vendors-edit' %}'>
-        <h4>Vendors</h4>
+        <!-- <h4>Vendors</h4> -->
+        <h4>{{ theme|get_fieldname:'vendors' }}</h4>
```

Comment:
I though if we are extending base.html then we won't need to explicitly load that tag, I thought you were talking about the inline templates (preparation_ajax etc...) as in them we don't extend from base.html.